src/bin/cratesfyi.rs

```diff
@@ -473,8 +473,8 @@ impl DatabaseSubcommand {
             }
 
             Self::AddDirectory { directory, prefix } => {
-                add_path_into_database(ctx.pool()?, &prefix, directory)
-                    .context("Failed to add directory into database")?;
+                add_path_into_database(&*ctx.storage()?, &prefix, directory)
+                    .expect("Failed to add directory into database");
```

Comment:
This looks like a bad rebase, can you change it to use `context()` instead of `expect()`?